Source/WebCore/Modules/reporting/ReportingObserver.idl

```diff
@@ -28,6 +28,7 @@
 typedef sequence<Report> ReportList;
 
 [
+    // ActiveDOMObject,
```

Comment:
I don't think this can be an active DOM object.
We're perpetually waiting for a new CSP violation, for example.
In effect, we need to "leak" any reporting observer whenever observe is called.